Correct grammar in `README.md`.

RokeJulianLockhart opened the pull request:
"to" should be "too".

RokeJulianLockhart committed:
Correct grammar in `README.md`

"to" should be "too".

JonasBernard commented on README.md:
```diff
@@ -4,7 +4,7 @@ Android app to open Moodle links of the TU Darmstadt in the official Moodle app.
 
 ## Disclaimer
-Neither is this an official project by the TU Darmstadt nor by Moodle. This is just started by me, a student at the TU Darmstadt, that has to much spare time.
+Neither is this an official project by the TU Darmstadt or Moodle. This is just started by me, a student at the TU Darmstadt, that has too much spare time.
```
The usage of 'nor' is correct, I guess.

But an even better phrasing would probably be:

This is neither an official project of TU Darmstadt nor of Moodle.

Or:

This is not an official project of either TU Darmstadt or Moodle.

beedell-roke replied:
@JonasBernard, I agree. I chose your latter suggestion.

RokeJulianLockhart committed:
Update `README.md`.

Incorporates https://github.com/JonasBernard/TU-Darmstadt-Moodle-Wrapper/pull/6/files#r2203506421.
